examples/presigned_post_policy.py

```diff
@@ -25,19 +26,36 @@
     secret_key="zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG",
 )
 
+# In play make sure my-bucket exist if not please create it
 policy = PostPolicy(
     "my-bucket", datetime.utcnow() + timedelta(days=10),
 )
-policy.add_starts_with_condition("key", "my/object/prefix/")
+# key=my/object/prefix/my-object-name should go in the policy and in the filled-in form
+# Same as in our Java example for the example to actually works.
+policy.add_starts_with_condition("key", "my/object/prefix/my-object-name")
```

Comment:
`add_starts_with_condition` should work with `my/object/prefix/`. If not, it is a server bug.